sdk/tests/conformance2/renderbuffers/framebuffer-test.html

```diff
@@ -188,6 +196,107 @@
   wtu.glErrorShouldBe(gl, gl.NO_ERROR,
             "detaching a renderbuffer from a framebuffer should succeed.");
 
+  // attach a 2d texture to read framebuffer, while no attachment to draw framebuffer.
+  gl.framebufferTexture2D(gl.READ_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, fbtex, 0);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "attaching a texture to read framebuffer should succeed.");
+  gl.getFramebufferAttachmentParameter(gl.FRAMEBUFFER,
+                                       gl.COLOR_ATTACHMENT0,
+                                       desktopGL.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
```

Comment:
FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING is part of the WebGL 2.0 spec so you could just as well get it from the "gl" object instead of the "desktopGL" object.